tests/conftest.py

```diff
@@ -5,23 +5,6 @@
 from workoutizer import settings as django_settings
 
 
-def pytest_addoption(parser):
```

Comment:
I'm not sure whether there is a meaningful way of combining this with my current approach of parametrizing the `webdriver` fixture using `request.param`.

@gotiniens What do you think?

Reply:
Nee this was purely meant for the temporary situation where we did not have full support for chrome. because running tests for chrome would fail the tests, which was not desirable yet.

Fixing for Chrome is now done, and we run tests for both browsers now, so this section is useless now